src/ir_BluestarHeavy.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright 2020 David Conran (crankyoldgit)
+/// @file
+/// @brief Support for BluestarHeavy protocol
+
+// Supports:
+//   Brand: Bluestar,  Model: TODO add device and remote
+
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRutils.h"
+
+// WARNING: This probably isn't directly usable. It's a guide only.
+
+// See https://github.com/crankyoldgit/IRremoteESP8266/wiki/Adding-support-for-a-new-IR-protocol
+// for details of how to include this in the library.
+const uint16_t kBluestarHeavyHdrMark = 4912;
+const uint16_t kBluestarHeavyBitMark = 465;
+const uint16_t kBluestarHeavyHdrSpace = 5058;
+const uint16_t kBluestarHeavyOneSpace = 1548;
+const uint16_t kBluestarHeavyZeroSpace = 572;
+const uint16_t kBluestarHeavyFreq = 38000;  // Hz. (Guessing the most common frequency.)
+//const uint16_t kBluestarHeavyBits = 104;  // Move to IRremoteESP8266.h
+//const uint16_t kBluestarHeavyStateLength = 13;  // Move to IRremoteESP8266.h
+const uint16_t kBluestarHeavyOverhead = 3;
+// DANGER: More than 64 bits detected. A uint64_t for 'data' won't work!
+
+
+#if SEND_BLUESTARHEAVY
+// Alternative >64bit function to send BLUESTARHEAVY messages
+// Function should be safe over 64 bits.
+/// Send a BluestarHeavy formatted message.
+/// Status: ALPHA / Untested.
```

Comment:
You indicate this works. So mark as BETA or STABLE / Expected to work, if it successfully controls your device.